src/DataFactory/DataFactoryV2/help/Set-AzDataFactoryV2.md

```diff
@@ -38,12 +99,78 @@ PS C:\> Set-AzDataFactoryV2 -ResourceGroupName "ADF" -Name "WikiADF" -Location "
     Tags              : {}
     Identity          : Microsoft.Azure.Management.DataFactory.Models.FactoryIdentity
     ProvisioningState : Succeeded
+    RepoConfiguration :
+
+PS C:\> Get-AzDataFactoryV2 -ResourceGroupName "ADF" -Name "WikiADF" | Set-AzDataFactoryV2 -AccountName msdata -RepositoryName ADFRepo -CollaborationBranch master -RootFolder / -ProjectName "Azure Data Factory"
```

Comment:
@cormacpayne this example you added should be in it's own example, so you'll need to add another example header for this piping scenario

Reply:
Done.